src/systems/tracked-controls.js

```diff
@@ -7,6 +7,7 @@ var utils = require('../utils');
  * It maintains a list with the available tracked controllers
  */
 module.exports.System = registerSystem('tracked-controls', {
+  schema: {alwaysAllowIdPrefixes: {type: 'array'}},
```

Comment:
originally for controllers that may have initial instability in gamepad api responses (classic example is when vive controllers first turn on, hand may be indeterminate, pose may be ill defined until confirmed within tracking volume)